backends/ebpf/psa/externs/ebpfPsaChecksum.cpp

```diff
@@ -85,4 +85,47 @@ void EBPFInternetChecksumPSA::processMethod(CodeBuilder* builder, cstring method
     }
 }
 
+void EBPFHashPSA::processMethod(CodeBuilder* builder, cstring method,
+                                const IR::MethodCallExpression * expr, Visitor * visitor) {
+    engine->setVisitor(visitor);
+
+    // Note: "Hash" extern has only one method "get_hash". Method "update" were introduced
+    // in order to be able to calculate hash and then assign it to a destination.
```

Comment:
Maybe this comment should be clarified: `... in order to calculate hash before assignment and assign it later in one statement`